### ISSUE TITLE
Missing |raw Twig filter in modular.html.twig

### ISSUE THREAD

**Issue by Fazarel** [pmoreno-rodriguez/grav-theme-editorial#4]
In modular.html.twig, in order to render html code,
```
        {{ module.content }}
```
should be :

```
        {{ module.content|raw }}
```

**Comment by pmoreno-rodriguez** [comment]
Thanks @Fazarel. I'll fix it for the next release.

**Comment by pmoreno-rodriguez** [comment]
Fixed in new version 1.0.4. Thanks
